interactions/api/models/guild.py

```diff
@@ -2271,6 +2272,160 @@ async def modify_auto_moderation_rule(
         )
         return AutoModerationRule(**res)
 
+    async def get_audit_logs(
+        self,
+        limit: Optional[int] = 100,
+        user_id: Optional[Union[User, int, Snowflake]] = MISSING,
+        action_type: Optional[Union[int, AuditLogEvents]] = MISSING,
+        before: Optional[Union[int, Snowflake]] = MISSING,
+    ) -> AuditLogs:
+        """
+        Gets the audit logs of the guild.
+
+        :param limit?: How many entries to get, default 100
+        :type limit?: Optional[int]
+        :param user_id?: User ID snowflake. filter the log for actions made by a user.
+        :type user_id?: Optional[Union[User, int, Snowflake]]
+        :param action_type?: The Type of the audit log action.
+        :type action_type?: Optional[Union[int, AuditLogEvents]]
+        :param before?: filter the log before a certain entry id.
+        :type before?: Union[int, Snowflake]
+        :return: The guild audit logs
+        :rtype: AuditLogs
+        """
+
+        _user_id = (
+            (user_id.id if isinstance(user_id, User) else user_id)
+            if user_id is not MISSING
+            else None
+        )
+        _before = before if before is not MISSING else None
+        _action_type = action_type if action_type is not MISSING else None
+
+        res = await self._client.get_guild_auditlog(
+            guild_id=int(self.id),
+            limit=limit,
+            before=_before,
+            user_id=_user_id,
+            action_type=_action_type,
+        )
+        return AuditLogs(**res)
+
+    async def get_latest_audit_log_action(
+        self,
+        of: Union[
```

Comment:
If it could be a tuple, why not *args?

Reply:
It can only be a Tuple of `(User, type)`, similar to members cache

Reply:
Why User? Why not Member?

Reply:
Because the param is named user_id and I just went with that, member works too tho, should I add it?

Reply:
Sure

Reply:
Already done